tutorials/models_mapping/README.txt

```diff
@@ -0,0 +1,12 @@
+Models and Mapping
+******************
+
+In SimPEG, the model represents the set of parameters that define our geology.
+For many problems, the model is a vector containing one or more physical
+property values for each cell in the mesh. However, we will see that there are
```

Comment:
Remove last sentence.

Reply:
I am thinking of a more direct approach where you list the topics/objectives of the tutorial, rather than a "story" type with specific use cases. Also, if we can find a way to hyperlink that would epic.. I really like this style (of course it's scipy... ):
https://docs.scipy.org/doc/scipy/reference/tutorial/interpolate.html

Reply:
I changed the last sentence. But I like taking the time to explain *what* something is. For some users, this may be the first time they have seen certain concepts. I don't want to write an essay but I do want to offer some context. For individual tutorials, I think there are clearer objectives.